lib/razorpay/utility.rb

```diff
@@ -0,0 +1,23 @@
+require 'razorpay'
+require 'openssl'
+
+# Helper functions are defined here
+module Razorpay
+  class Utility
+    def self.validate_payment_signature(order_id, payment_id, signature)
```

Comment:
But you are still accepting order_id, no? My point is for the doc to be clear that the order_id passed to this method should not be tainted